mantidproject/mantid
# Fix highlight multiple curves incorrectly enables the errorbar tab

**Description of work.**
This fixes an issue where the errorbar tab was wrongly enabled if you selected multiple curves by holding down the mouse button. This was due to the following:

Holding down the mouse button fires the following signals:
Selection changed -> Disables tab
Row changed -> enables the tab

Whereas shift clicking multiple selection does
Row changed -> enables tab
Selection change -> Disables tab

The fix is to stop the row changed signal enabling the tab if we have multiple selections





**To test:**
```
ws = CreateSampleWorkspace()
plotSpectrum(ws, [0,1,2,3])
```

- Go to figure options curves tab
- Select multiple curves using whatever method you wish
- The error bar tab should remain disabled
- likewise if we are only selecting a single curve the error bar tab should be enabled.



Fixes #30593. 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
